lib/common/utils/mixin/callback_after_build.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+/// A mixin that can be used with a [StatefulWidget] to register a callback.
+/// function [callbackAfterBuild] to be called after the widget is built.
```

Comment:
[nits]
公式で、概要と詳細のコメントは１行空けた方が好ましいとされているため、そのように修正した方が良さそうです 🙆‍♂️ 

https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

```suggestion
/// A mixin that can be used with a [StatefulWidget] to register a callback.
///
/// function [callbackAfterBuild] to be called after the widget is built.
```